checks/system/win32.py

```diff
@@ -201,66 +201,21 @@ class Cpu(Check):
     def __init__(self, logger):
         Check.__init__(self, logger)
 
-        # Sampler(s)
-        self.wmi_sampler = WMISampler(
-            logger,
-            "Win32_PerfRawData_PerfOS_Processor",
-            ["Name", "PercentInterruptTime"]
-        )
-
         self.counter('system.cpu.user')
         self.counter('system.cpu.idle')
-        self.gauge('system.cpu.interrupt')
         self.counter('system.cpu.system')
+        self.counter('system.cpu.interrupt')
```

Comment:
Basically an old-style-check `counter`s is the equivalent of a `rate`s of a new-style check. It's still stored as a gauge in the backend, so there's no backwards-compatibility issue.

We compute a `rate` here because `psutil.cpu_times` returns for `interrupt` the same kind of value as for the other cpu values, i.e. the total time the cpu(s) spent in the interrupt state since startup, so to get a percentage from this we need to compute the rate divided by the number of cpus and multiply that by 100.